initializr-service/src/main/resources/application.yml

```diff
@@ -1253,6 +1261,38 @@ initializr:
           versionRange: "[1.3.0.RELEASE,2.0.0.M1)"
           groupId: org.springframework.boot
           artifactId: spring-boot-actuator-docs
+    - name: Spring Cloud GCP
+      bom: spring-cloud-gcp
+      versionRange: 2.0.0.RC2
+      content:
+        - name: GCP Support
+          id: cloud-gcp
+          description: Spring Boot support to use GCP services through Spring Cloud
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-gcp-starter
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-gcp/docs/1.0.0.M2/reference/htmlsingle/
```

Comment:
Since GCP support combined with other dependencies will allow you to get dedicated support, perhaps a "getting started" section in the dev guide would be better? You can't really talk about initializr since the support hasn't been merged but we can update that later on

Reply:
I can add a link to https://cloud.google.com/java/docs/reference/spring if that's better.

If this requires adding a new section to the reference guide, I'd rather add it as an issue in our tracker and later change the link here.